.travis.yml

```diff
@@ -33,10 +34,13 @@ before_install:
       brew install yajl;
       brew install augeas --HEAD;
       brew install dbus;
-      brew install qt5;
+      # Building Qt5 currently takes too much time.
```

Comment:
Why are you trying to build qt5, aren't there any binaries (with dbus)?

If they do not exist, we should simply opt-out dbus (see like it is done for Svg + a ifdef in src/treeviewmodel.cpp to not include the header file and body of TreeViewModel::connectDBus).